From 70c67a459f94f65e33e6eaf5ca2120dd2dc8de78 Mon Sep 17 00:00:00 2001
Message-Id: <70c67a459f94f65e33e6eaf5ca2120dd2dc8de78.1527783193.git.Jim.Somerville@windriver.com>
From: eric zhang <eric.zhang@windriver.com>
Date: Wed, 14 Mar 2018 14:56:21 -0400
Subject: [PATCH] Add build dependencies and package versioning for TiS format

Force disable python3 to avoid random build failure due to go to
python3 path.

Signed-off-by: eric zhang <eric.zhang@windriver.com>
Signed-off-by: Jim Somerville <Jim.Somerville@windriver.com>
Signed-off-by: Shuicheng Lin <shuicheng.lin@intel.com>
---
 SPECS/rdma-core.spec | 13 +++++++++++--
 1 file changed, 11 insertions(+), 2 deletions(-)

diff --git a/SPECS/rdma-core.spec b/SPECS/rdma-core.spec
index 1f8af03..bd0e7a5 100644
--- a/SPECS/rdma-core.spec
+++ b/SPECS/rdma-core.spec
@@ -1,3 +1,9 @@
+%if "%{_tis_build_type}" == "rt"
+%define bt_ext -rt
+%else
+%undefine bt_ext
+%endif
+
 %{!?cmake: %global cmake cmake}
 %{!?make_jobs: %global make_jobs make VERBOSE=1 %{?_smp_mflags}}
 %{!?cmake_install: %global cmake_install DESTDIR=%{buildroot} make install}
@@ -15,7 +21,7 @@
 
 Name: rdma-core
 Version: 47mlnx1
-Release: 1%{?dist}.47329
+Release: 1.47329%{?_tis_dist}.%{tis_patch_ver}
 Summary: RDMA core userspace libraries and daemons
 Group: System Environment/Libraries
 
@@ -57,11 +63,14 @@ BuildRequires: python3-devel
 BuildRequires: python3-Cython
 %endif
 %else
-BuildRequires: python
+BuildRequires: python3
 %endif
 %if 0%{?fedora} >= 21
 BuildRequires: perl-generators
 %endif
+BuildRequires: cmake
+BuildRequires: libnl3-devel
+BuildRequires: mlnx-ofa_kernel%{?bt_ext}-devel
 
 Requires: dracut, kmod, systemd
 # Red Hat/Fedora previously shipped redhat/ as a stand-alone
-- 
2.17.1

